ctdcal/process_ctd.py

```diff
@@ -1937,23 +1938,16 @@ def export_ct1(df, ssscc_list):
                     print(col + " missing, filling with -999s")
                     df[col] = -999
 
-    # load other necessary params
-    # TODO: this will change when config.py improves, find different method
-    expocode = cfg.cruise["expocode"]
-    section_id = cfg.cruise["sectionid"]
-    ctd = cfg.ctd_serial
-    out_dir = cfg.directory["pressure"]
-    p_col = 'CTDPRS'
-    stacst_col = 'SSSCC'
-    logFile = cfg.directory["logs"] + 'cast_details.csv'
-
-    df[stacst_col] = df[stacst_col].astype(str).copy()
-
+    df["SSSCC"] = df["SSSCC"].astype(str).copy()
     cast_details = pd.read_csv(cfg.directory["logs"] + "cast_details.csv", dtype={"SSSCC": str})
-    depth_df = pd.read_csv(cfg.directory["logs"] + 'depth_log.csv').dropna()
-    manual_depth_df = pd.read_csv(cfg.directory["logs"] + 'manual_depth_log.csv')
-    full_depth_df = pd.concat([depth_df,manual_depth_df])  # TODO: update from STNNBR to SSSCC
-    full_depth_df.drop_duplicates(subset='STNNBR', keep='first',inplace=True)
+    depth_df = pd.read_csv(cfg.directory["logs"] + 'depth_log.csv', dtype={"SSSCC": str}).dropna()
+    try:
+        manual_depth_df = pd.read_csv(cfg.directory["logs"] + 'manual_depth_log.csv', dtype={"SSSCC": str})
+    except FileNotFoundError:
+        manual_df = depth_df.copy()  # write manual_depth_log as copy of depth_log
+        manual_df.to_csv(cfg.directory["logs"] + 'manual_depth_log.csv', index=False)
```

Comment:
There should be a logger/print statement letting you know you hit a `FileNotFoundError` so you don't get confused later on with behavior you don't expect.